content/en/docs/install-operate/production-performance.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Falco Performance in Production 
+description: Adopt best practices when checking Falco performance in production
+linktitle: Falco Performance in Production
+weight: 45
+---
+
+If something seriously goes wrong during Falco deployment, it's usually noticeable immediately. On a longer time scale, continuous performance monitoring and quality assurance, driven by the right metrics, ensure Falco functions as expected 24/7.
+
+As a security tool, Falco requires checking for a healthy deployment on multiple dimensions:
+
+- Resource Utilization and System Impact: Aim to achieve the lowest possible compute overhead while ensuring the appropriate monitoring scope.
+- Disruption / Upgrades: Ensure minimal downtime and avoid interruptions to the service, minimizing restarts.
+- Data Quality Assurance: Verify that Falco outputs contain the desired quality with little to no missing data.
+- End-to-end testing of data pipeline: Ensure alerts reach their end destination as quickly as possible.
+- Security Monitoring Capabilities: Adopting the right Falco rules and resilience to bypasses by attackers.
+
+The Falco Project provides guidance on some of these aspects, and there are ongoing long-term efforts, including a [partnership](https://github.com/falcosecurity/falco/issues/2435) with the CNCF TAG Environmental Sustainability, aimed at enhancing Falco's performance and assessing its impact on the system. These efforts are intended to make it easier for adopters to assess the actual impact on their systems, enabling them to make informed decisions about the cost-security monitoring tradeoffs.
+
+Other aspects, such as data quality assurance and security monitoring capabilities, are continually improved upon, and with each Falco release, Falco becomes better and more efficient.
+
+## Resource Utilization and System Impact
```

Comment:
@FedeDP since we worked together on the adaptive syscalls selection blog post, would you have feedback?